Library/Homebrew/extend/blank.rb

```diff
@@ -48,44 +46,32 @@ class NilClass
   #
   #   nil.blank? # => true
   sig { returns(TrueClass) }
-  def blank?
-    true
-  end
+  def blank? = true
```

Comment:
A bit of Ruby 3 cleanup while were dealing with `blank?`. (Since these work like constants, I quite prefer this style, but lmk if this inappropriate for the scope of this PR.)